### PR TITLE
deviceauth migrate --tenant

### PR DESCRIPTION
**Pull request by mchalski**
https://tracker.mender.io/browse/MEN-1244

@maciejmrowiec @mendersoftware/rndity 

acceptance tests finally fixed, see last commit

note that coverage is down - I didn't want to unit test the command, lots of effort, acceptance tests are provided instead.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -67,6 +69,18 @@ func doMain(args []string) {
 
 			Action: cmdServer,
 		},
+		{
+			Name:  "migrate",
+			Usage: "Run migrations and exit",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "tenant",
+					Usage: "Tenant ID (optional).",
```
Comment (maciejmrowiec) [inline main.go]: if I get this correctly `./deviceauth migrate` will migrate all dbs, `./deviceauth migrate --tenanant 123` just a single one right ?

Comment (bboozzoo) [reply main.go]: No, `migrate` will migrate the default DB, `migrate --tenant 123` will migrate only the DB owned by tenant 123.

File: tests/tests/test_cli.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        print('found migration:', mi)
+        assert mi
+
+    @staticmethod
+    def verify(cli, mongo, dbname, version):
+        TestMigration.verify_db_and_collections(mongo, dbname)
+        TestMigration.verify_migration(mongo[dbname], version)
+
+
+@pytest.mark.last
```
Comment (bboozzoo) [inline tests/tests/test_cli.py]: Can you leave a note as to why this must run last?

Comment (mchalski) [reply tests/tests/test_cli.py]: done

**Pull request merged**